micro-rdk-server/esp32/main.rs

```diff
@@ -70,30 +72,38 @@ mod esp32 {
 
         let mut registry = Box::<ComponentRegistry>::default();
         register_example_modules(&mut registry);
+
         #[cfg(feature = "qemu")]
         let storage = { RAMStorage::new() }; //NVSStorage::new("nvs").unwrap();
         #[cfg(not(feature = "qemu"))]
         let storage = { NVSStorage::new("nvs").unwrap() };
+
         // At runtime, if the program does not detect credentials or configs in storage,
         // it will try to load statically compiled values.
 
         if !storage.has_wifi_credentials() {
+            log::warn!("no wifi credentials were found in storage");
+
             // check if any were statically compiled
             if SSID.is_some() && PASS.is_some() {
-                log::info!("Storing static values from build time wifi configuration to NVS");
+                log::info!("storing static values from build time wifi configuration to storage");
                 storage
                     .store_wifi_credentials(&WifiCredentials::new(
                         SSID.unwrap().to_string(),
                         PASS.unwrap().to_string(),
                     ))
-                    .expect("Failed to store WiFi credentials to NVS");
+                    .expect("failed to store WiFi credentials to storage");
             }
         }
 
-        if !storage.has_robot_configuration() {
+        if !storage.has_robot_credentials() {
+            log::warn!("no machine credentials were found in storage");
```

Comment:
I've tried to consistently replace `robot` with `machine` in the logs I've updated.